### PR TITLE
Mayland Blocks: bring up-to-date with GB 10.3

### PR DESCRIPTION
**Pull request by pbking**
This change is for BOTH Seedlet Blocks and Mayland Blocks active development.  

Both of them are BCB children and benefit from similar changes while under active development.

### CODE REVIEW

File: mayland-blocks/functions.php [hunk 1]
```diff
@@ -65,9 +65,6 @@ function mayland_blocks_scripts() {
 	// Enqueue front-end styles.
 	wp_enqueue_style( 'mayland-blocks-styles', get_stylesheet_uri() );
 
-	// Enqueue front-end styles.
-	wp_enqueue_style( 'mayland-blocks-block-styles', get_stylesheet_directory_uri() . '/assets/alignments.css' );
-
 	// Enqueue Google fonts, if necessary.
 	wp_enqueue_style( 'mayland-blocks-fonts', mayland_blocks_fonts_url(), array(), null );
```
Comment (scruffian) [inline mayland-blocks/functions.php]: I think we can remove this if and load them from theme.json

File: seedlet-blocks/style.css [hunk 2]
```diff
@@ -28,28 +28,42 @@ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
...
-/* Make sure full-width items in entry content show up as full-width. */
-.wp-block-group.site-content .wp-block-group__inner-container .entry-content {
-	max-width: calc(100% + (2 * var(--global--spacing-horizontal)));
-	width: calc(100% + (2 * var(--global--spacing-horizontal)));
-	margin-left: calc(-1 * var(--global--spacing-horizontal));
-	margin-right: auto;
-}
\ No newline at end of file
+/**
+ * Author bio
```
Comment (scruffian) [inline seedlet-blocks/style.css]: This should go into a sass file

**Comment by scruffian** [comment]
I think we can remove these lines from Mayland Blocks, since they will happen in the Parent:
```
		// Alignwide and alignfull classes in the block editor.
		add_theme_support( 'align-wide' );
		// Adding support for responsive embedded content.
		add_theme_support( 'responsive-embeds' );
		// Add support for experimental link color control.
		add_theme_support( 'experimental-link-color' );
		// Add support for editor styles.
		add_theme_support( 'editor-styles' );
```

File: seedlet-blocks/style.css (hunk 2 above)
Comment (pbking) [reply seedlet-blocks/style.css]: https://github.com/Automattic/themes/pull/3575

**Comment by scruffian** [comment]
The Mayland blocks social menu looks weird:
Screenshot 2021-03-30 at 17 44 16

File: seedlet-blocks/block-templates/index.html [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
-<!-- wp:post-title {"isLink":true} /-->
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:post-title {"isLink":true} /--></div>
+<!-- /wp:group -->
 
-<!-- wp:post-content /-->
+<!-- wp:post-content {"layout":{"inherit":true}} /-->
 
-<!-- wp:post-date {"fontSize":"tiny"} /-->
+<!-- wp:template-part {"slug":"post-meta","theme":"seedlet-blocks","layout":{"inherit":true}} /-->
```
Comment (scruffian) [inline seedlet-blocks/block-templates/index.html]: We should remove the theme slug, and add the template-part to the theme json.

Comment (scruffian) [reply seedlet-blocks/block-templates/index.html]: Done in a2b666fc421dabf22792d1884c76a8f9b5006c12

**Pull request merged**